src/NServiceBus.ContainerTests/When_registering_components.cs

```diff
@@ -131,42 +97,6 @@ public void Setter_dependencies_should_be_supported_when_resolving_interfaces()
             }
         }
 
-
-        [Test]
-        public void Setter_dependencies_should_be_supported()
```

Comment:
I always say "the only good code is deleted code". I also always say "property injection is :hankey:." Especially the way we do it with no property decoration. That behaviour is a horrible code smell and non-obvious and has a bunch of edge cases that can get awkward.

So I'm :+1: for deleting the property injection tests and all code from the containers.

If someone wants to use a container's property injection, they're welcome to but we don't have to encourage them to do so. Pit of success and all that.